Actor Replication Net Relevancy (#1348)
spatialos/UnrealGDK: CHANGELOG.md

- Visual Studio 2019 is now supported.
- Added toolbar and commandlet options to delete the schema database.
- Added a check for schema and snapshot before attempting to start a local deployment. If either are missing then an error message will be displayed.

> - Added optional net relevancy check in replication prioritization. If enabled, an actor will only be replicated if at least one player controller falls within its net view cull distance or if the actor is set to be always relevant

Reviewer: ```suggestion
- Added optional net relevancy check in replication prioritization. If enabled, an actor will only be replicated if IsNetRelevantFor is true for one of the connected client's views.
```
